exercises/03-exercise.py

```diff
@@ -0,0 +1,121 @@
+''' 
+Author: Jorge Diaz Jr
+Exercise 3
+'''
+import pandas as pd
+import matplotlib.pyplot as plt
+import numpy as np
+
+file_path = "/home/jdiaz/projects/data-monitoring/data/xas_data/xas_test_328.txt"
+
+def get_header(file_path):
+	'''
+	Open file , reads file in, splits line apart if it starts with #, 
+	one piece is the key, the other piece is the value
+	e.g. # Year: 2018
+	gets stored in a dictionary such that hdr_dict['Year'] = '2018'
+	returns the dictionary hdr_data, and returns a list of all the lines
+	such that the lines do NOT start with # symbol
+	'''
+	hdr_data = {}
+	key_value = []
+	info_data = []
+
+	#opens file and reads each line, extracting key and value pair if line starts with '#'
+	with open(file_path) as file:
+		for line in file:
+			if line.startswith("#"):
+				if len(line) > 1:
+					key_value = line.split(":", 1)
+					if len(key_value) == 2:
+						key = key_value[0].replace("#","").strip()
+						value = key_value[1].replace("\n","").strip()
+						hdr_data[key] = value
+					elif len(key_value) == 1:
+						continue	
+			if not line.startswith("#"):
+				info_data.append(line.strip().split())
+
+	return hdr_data, info_data
+
+def get_column_names(file_path):
```

Comment:
the column is already obtained by `get_header` correct? 
`hdr_data.keys()` can also give this information.
if you need it to be ordered, you can used a special type of dictionary: `OrderedDict`
```py3
In [1]: from collections import OrderedDict

In [2]: d = OrderedDict()

In [3]: d['a']= 1

In [4]: d
Out[4]: OrderedDict([('a', 1)])

In [5]: d.keys()
Out[5]: odict_keys(['a'])

In [6]: list(d.keys())
Out[6]: ['a']
```

this might help remove an extra function.